src/docfx/config/ops/OpsConfigAdapter.cs

```diff
@@ -129,6 +129,7 @@ private async Task<string> GetBuildConfig(Uri url)
                     $"{MetadataSchemaApi}{metadataServiceQueryParams}",
                 },
                 xref = xrefMaps,
+                isReference = docset.use_template,
```

Comment:
What if `use_template` translate to sth. like `monikerRange { '/': '*' }'` ? I didn't quite like this `isReference` flag.

Reply:
#5714 to support `*` in moniker range parser

Reply:
@928PJY had a concern that, if the user misuse `*` a non-versioning repo, then every article will output `monikers` with all the monikers within Docs, which is more than 1000

Reply:
So `*` only be used for checking if it is inside a valid moniker range?

Reply:
how to limit the usage?

Reply:
A better name should suffice.